minio_service/minio_client.py

```diff
@@ -0,0 +1,80 @@
+from config import *
```

Comment:
please don't do this, import only what you need

Reply:
done